core/controllers/admin.py

```diff
@@ -985,12 +979,6 @@ def put(self):
 
         self.render_json({})
 
-
-class ContributorUsersListHandler(base.BaseHandler):
-    """Handler to show users with contribution rights."""
-
-    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
-
     @acl_decorators.can_access_admin_page
     def get(self):
```

Comment:
This get method corresponds to multi users data. I think the better get call for the ContributionRightsHandler handler will be the get method in `ContributionRightsDataHandler`.

@vojtechjelinek wdyt? Do you have any suggestions?

Reply:
Yep, I guess that would be a better place.

Reply:
We already have a get function in ContributionRightsDataHandler right so how do I proceed?

Reply:
After replacing them.
All backend tests were passing but frontend fails what else do I have to update here?

Reply:
@FareesHussain You'll have to change the url in the frontend to support this new API.